.env

```diff
@@ -1,13 +1,13 @@
-TESTVAR= {{ credentials.TESTVAR }}
-PORT = {{ credentials.PORT }}
-CRED_URL = {{ credentials.CRED_URL }}
-DID_URL = {{ credentials.DID_URL }}
-SCHEMA_URL = {{ credentials.SCHEMA_URL }}
-KEYCLOAK_REALM_ID= {{ credentials.KEYCLOAK_REALM_ID }}
-KEYCLOAK_CLIENT_ID= {{ credentials.KEYCLOAK_CLIENT_ID }}
-KEYCLOAK_CLIENT_SECRET= {{ credentials.KEYCLOAK_CLIENT_SECRET }}
-KEYCLOAK_URL= {{ credentials.KEYCLOAK_URL }}
-REGISTRY_URL= {{ credentials.REGISTRY_URL }}
-TELEMETRY_URL= {{ credentials.TELEMETRY_URL }}
+TESTVAR= development environment
```

Comment:
discard all changes in .env file and then commit code again

Reply:
resolved